crates/wash-cli/src/up/mod.rs

```diff
@@ -711,6 +725,21 @@ fn is_process_running(pid: &str) -> bool {
     }
 }
 
+/// Parses max_execution_time input string to return a duration string.
+fn parse_duration(s: &str) -> Result<String, String> {
```

Comment:
If we do manual parsing, let's return `anyhow::Result<Duration>`

Reply:
I don't think we should continue the manual parsing, humantime::Duration is much more flexible 
input like `2h45min22s` gets auto-parsed into `Duration` which we can directly push into `host_config` as either milliseconds or seconds depending on which one is more granular.